WooCommerce/Classes/POS/ViewModels/ItemListViewModel.swift

```diff
@@ -10,6 +10,8 @@ final class ItemListViewModel: ItemListViewModelProtocol {
     @Published private(set) var isHeaderBannerDismissed: Bool = false
     @Published var showSimpleProductsModal: Bool = false
 
+    @Published private(set) var currentPage: Int = 0
```

Comment:
❓ Is this variable public just for testing? I think generally we keep variables private whenever possible. Maybe the assertions can be made on the `itemProvider.providePointOfSaleItems`'s `pageNumber`?

Reply:
It seems it is 😅 , I'm going to leave it like this for now based on the next changes described here https://github.com/woocommerce/woocommerce-ios/pull/14195/files?diff=unified&w=0#r1813903775 as I think that's what we're going to test, otherwise I'll make it fully private.

> Maybe the assertions can be made on the itemProvider.providePointOfSaleItems's pageNumber?

I'm curious, could you clarify how could we test this bit? Do you mean for example asserting if we get the correct `simulatedResponse` for different pages?

Reply:
> I'm curious, could you clarify how could we test this bit? Do you mean for example asserting if we get the correct simulatedResponse for different pages?

In `ItemListViewModelTests`, it's already using a mock item provider `MockPOSItemProvider`. We can potentially add a variable to `MockPOSItemProvider` for the latest page number, set by `providePointOfSaleItems`, and the test cases can have assertions on this page number property.